terracommon/datastore/models.py

```diff
@@ -32,9 +34,10 @@ class Meta:
 
 
 def related_document_path(instance, filename):
+    filename = Path(filename)
     return (f'documents/'
             f'{instance.content_type.app_label}_{instance.content_type.model}'
-            f'/{instance.object_id}/{instance.key}')
+            f'/{instance.object_id}/{instance.key}{filename.suffix}')
```

Comment:
What if there is no suffix? will it be suffixed by None ?

Reply:
```
>>> from pathlib import Path
>>> Path('prout').suffix
''
```